src/dbus_api.rs

```diff
@@ -241,6 +296,11 @@ fn create_pool(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
             let dbus_context_clone = dbus_context.clone();
             let object_path: dbus::Path = create_dbus_pool(dbus_context_clone);
             let (rc, rs) = code_to_message_items(StratisErrorEnum::STRATIS_OK);
+
+            let dbus_context_clone = dbus_context.clone();
```

Comment:
Yeah a clone just bumps the refcount on the Rc.

The style comment I was making was just that we could pass a ref to the Rc<RefCell<DbusContext>> into create_dbus_pool and then it could clone internally, and that that would be slightly tidier.